.gitignore

```diff
@@ -3,3 +3,4 @@
 lib/
 node_modules
 usage.txt
+.idea/
```

Comment:
If we add it to the repo, then we don't have to worry about some git noob adding their whole IDE config folder in a PR.

Is this really worth worrying about?